sdk/core/core-compat/src/policies/keepAlivePolicy.ts

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import {
+  PipelinePolicy,
+  PipelineRequest,
+  SendRequest,
+  PipelineResponse,
+} from "@azure/core-rest-pipeline";
+
+export const keepAlivePolicyName = "KeepAlivePolicy";
+
+/**
+ * A policy factory for setting the Accept header to ignore odata metadata
```

Comment:
Doc comment seems like it was copied from somewhere else.

Reply:
Modified it